### PR TITLE
feat: Make it possible to start UIA2 server without an app or a package id

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -190,12 +190,14 @@ class AndroidUiautomator2Driver extends BaseDriver {
         // find and copy, or download and unzip an app url or path
         this.opts.app = await this.helpers.configureApp(this.opts.app, [APK_EXTENSION, APKS_EXTENSION]);
         await this.checkAppPresent();
-      } else if (this.appOnDevice) {
+      } else if (this.opts.appPackage) {
```
Comment (imurchie) [inline lib/driver.js]: Does this driver now take the `appPackage` from the `app` capability if it is there? Why not continue using `this.appOnDevice`?

Comment (mykola-mokhnach) [reply lib/driver.js]: `appOnDevice` was a blind copy paste from AndroidDriver, which is not even assigned anywhere in UIA2

Comment (mykola-mokhnach) [reply lib/driver.js]: > Does this driver now take the appPackage from the app capability if it is there?

It was always like that:

```
    // get appPackage et al from manifest if necessary
    const appInfo = await helpers.getLaunchInfo(this.adb, this.opts);
    // and get it onto our 'opts' object so we use it from now on
    Object.assign(this.opts, appInfo || {});
```

Comment (imurchie) [reply lib/driver.js]: Ah. Getters don't get assigned like functions do. Alas.

**Pull request merged**